### ISSUE TITLE
About the training on ImageNet-Caltech

### ISSUE THREAD

**Issue by Solacex** [thuml/Universal-Domain-Adaptation#8]
Hello,

Could you share with us the following details on ImageNet-Caltech? 
batch-size and learning rate.

**Comment by Solacex** [comment]
And if the model removes the bottleneck like https://github.com/thuml/SAN
